e2e-tests/create-large-test-files.sh

```diff
@@ -1,4 +1,4 @@
-dd if=/dev/zero of=cypress/fixtures/appeal-statement-valid-max-size.png bs=104857600 count=1
-dd if=/dev/zero of=cypress/fixtures/appeal-statement-invalid-too-big.png bs=104857601 count=1
-dd if=/dev/zero of=cypress/fixtures/upload-file-valid-max-size.png bs=104857600 count=1
-dd if=/dev/zero of=cypress/fixtures/upload-file-invalid-too-big.png bs=104857601 count=1
+dd if=/dev/zero of=cypress/fixtures/appeal-statement-valid-max-size.png bs=52428800 count=1
```

Comment:
Can we put `$FILE_UPLOAD_MAX_FILE_SIZE_BYTES` so it's driven by the envvars? The `+1` is slightly more complex, but it means it's config driven

Reply:
Good point. I will make it conditional so that if the env var is available then it will be used but if not a default value of 50Mb will be used.

Reply:
That's seems fair enough. From looking how it's invoked, the `.github/workflows/e2e-tests-on-PR.yml` and `.github/workflows/publish-cucumber-on-merge.yml` files may need updating to add the `FILE_UPLOAD_MAX_FILE_SIZE_BYTES` envvar.

Personally, so long as the test is in and works, the actual size is irrelevant to the configuration.

This PR won't actually change the max file size in production. That's controlled from `charts/app/values.yaml` (the `documentServiceApi.config.upload.maxSize` value).

Reply:
All sounds good. I have implemented the change.